ide-support/revsaveasstandalone.livecodescript

```diff
@@ -731,11 +731,45 @@ command revSaveAsMacStandalone pStack, pRestoreFileName, pFolder, @xStandalonePa
       __RestoreStackFromTemporaryLocation pStack, pRestoreFileName, tOriginalFileName
    end repeat
    
+   if the platform is "MacOS" then
+      local tAppBundle
+      set the itemDel to slash
+      put item 1 to -4 of tStandalonepath into tAppBundle
+      
+      replace space with "\ " in tAppBundle
+      clearExtendedAttributes tAppBundle
+      performAdHocCodesign tAppBundle
+   end if
+   
    revStandalonePostBuild pStack, pFolder
    
    return empty
 end revSaveAsMacStandalone
 
+private command performAdHocCodesign pAppBundle
```

Comment:
@livecodepanos probably better for `pAppBundle` to be either unescaped or changed to something like `pEscapedAppBundlePath` or something so someone doesn't come along later and add quotes or something.